ResearchKit/Common/ORKHelpers.h

```diff
@@ -34,6 +35,37 @@
 #import <ResearchKit/ORKErrors.h>
 
 
+#if ( defined(ORK_LOG_LEVEL_NONE) && ORK_LOG_LEVEL_NONE )
+#  undef ORK_LOG_LEVEL_DEBUG
+#  undef ORK_LOG_LEVEL_WARNING
+#  undef ORK_LOG_LEVEL_ERROR
+#endif
+
+#if ( !defined(ORK_LOG_LEVEL_NONE) && !defined(ORK_LOG_LEVEL_DEBUG) && !defined(ORK_LOG_LEVEL_WARNING) && !defined(ORK_LOG_LEVEL_ERROR) )
+#  define ORK_LOG_LEVEL_WARNING 1
```

Comment:
Learned:)
You are right, the first party apps are leaving log on for `warnings` and `errors`.
So let's make it explicit in the project file?

Debug | ORK_LOG_LEVEL_WARNING=1
Release | ORK_LOG_LEVEL_WARNING=1